### DISCUSSION TITLE
How to delete unfinished plots

### DISCUSSION THREAD

**Discussion by FrancescoSaverioZuppichini** [guydavis/machinaris#231] [Q&A] [answered]
Hi guys,

Assuming I've stoped and restarted the machinaris container, how do I delete the unfinished plots?

Thanks!

Francesco

**Comment by guydavis** [comment, marked as answer]
Good day!  Machinaris will automatically delete stale *.tmp files in the plotting locations (both tmp and tmp2) when starting up the next job.  No need to manually do this cleanup.

**Reply by FrancescoSaverioZuppichini** [reply]
Amazing, thanks you :)
